pytroll_collectors/scisys.py

```diff
@@ -520,15 +534,7 @@ def receive_from_zmq(host, port, station, environment, excluded_platforms,
             logger.debug("to_send: %s", str(to_send))
             if to_send is None:
                 continue
-            if topic_postfix is not None:
-                subject = "/".join(("", to_send['format'],
-                                   to_send['data_processing_level'],
-                                   topic_postfix))
-            else:
-                subject = "/".join(("", to_send['format'],
-                                   to_send['data_processing_level'],
-                                   station, environment,
-                                   "polar", "direct_readout"))
+            subject = _get_subject_from_msg2send(to_send, topic_postfix)
```

Comment:
According to Codecov, this line where the subject forming (= the change made here) happens isn't covered by the tests.